observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -71,10 +69,7 @@ Some features in this release are currently in Technology Preview. These experim
 link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
 
 [id="ebpf-manager-operator-with-network-observability_{context}"]
-==== eBPF Manager Operator with Network Observability
-
-:FeatureName: eBPF Manager Operator with Network Observability
-include::snippets/technology-preview.adoc[]
+==== eBPF Manager Operator with Network Observability (Technology Preview)
```

Comment:
Do we need the _(Technology Preview)_, where the entry is under _Technology Preview features_?